lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    rvtu: 'Votos recibidos/ usuarios totales',
+    reply: 'Repetir',
+    safariBrowserIssue: 'Su navegador Safari {virtualclass1} no es compatible. Es aconsejable usar Chrome y Firefox',
+    Save: 'Grabar',
+    savesession: '¿Quieres guardar la sesión actual?',
+    savesessionTechSupport: "¿Quieres guardar la sesión actual? <br /> Una vez que descargue la sesión, las actualizaciones no estarán disponibles al volver a cargarla.",
+    screensharealready: 'La pantalla se está compartiendo.',
+    ScreenShare: 'Compartir pantalla',
+    screensharemsg: 'La pantalla de arriba se está compartiendo.',
+    screensharenotsupport: 'Pantalla compartida no soportada',
+    screensharedenied: 'El usuario rechazó compartir su pantalla',
+    second: 'Segundo',
+    selfview: 'Deshabilitar vista pública',
+    SetTimer: 'Establecer tiempo',
+    sesseionkeymissing: 'Falta la KEY de la interfaz', // from javascript
+    sentPackets: 'Enviar <br/><span>Paquetes</span>',
+    // 'sessionend': "Close session.",
+    SessionEnd: 'Sesión finalizada',
+    sessionexpired: 'La sesión ha expirado',
+    sessionendmsg: 'La sesión ha sido cerrada. Puede cerrar su navegador.',
+    setting: 'Configuración',
+    Searchuser: 'Buscar usuario',
+    sendMessage: 'Enviar mensaje',
+    share: 'Compartir',
+    ShareVideo: 'Compartir vídeo',
+    shareAnotherYouTubeVideo: 'Compartir otro vídeo de YouTube',
+    SharePresentation: 'Compartir presentación',
+    SharePresentationdbHeading: 'Panel de Presentación',
+    sharetoall: 'Habilitar vista pública',
+    Showresulttostudents: 'Mostrar resultados de los alumnos',
+    sitePoll: 'Encuesta nivel del centro',
+    Skip: 'Cerrar',
+    someproblem: 'Hubo algún problema al cargar el archivo. Inténtelo de nuevo.',
+    SourceUnavailableError: 'Por favor, asegúrese de que no se esté utilizando la misma webcam en otros navegadores o aplicaciones.',
+    // 'SourceUnavailableError' : 'Otra aplicación podría estar usando su webcam. Cierre las demás aplicaciones que puedan estar utilizándola.',
+    speakerTest: 'Si puedes escuchar música, los altavoces están funcionando.',
+    Spoll: 'Encuesta del centro',
+    ssBtn: 'Deja de compartir',
+    ssStop: 'Dejar de compartir pantalla',
+    startnewsession: '¿Seguro qué quieres finalizar esta sesión?',
+    status: 'Estado',
+    stdPublish: 'stdPublicado',
+    stdscreenshare: '¿Quieres compartir tu pantalla?',
+    studentAudEnable: 'Audio del alumno activado',
+    studentSafariBrowserIssue: 'Su navegador {virtualclass1} {virtualclass2} no puede compartir su webcam con otros usuarios, Congrea solo es compatible con Chrome y Firefox.',
+    supportDesktop: 'La clase virtual no es compatible con dispositivos móviles. Es aconsejable usar un pc o portátil.',
+    supportDesktopOnly: 'Lo sentimos, el modo de presentación solo es compatible con pcs o portátiles. Los dispositivos móviles no son totalmente compatibles.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 160. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)